docs/advanced_usage.rst

```diff
@@ -306,22 +306,22 @@ Before upgrading, we recommend resolving any deprecation warnings raised in your
 
 .. _upgrade-0.x:
 
-0.x Release Line
-^^^^^^^^^^^^^^^^
+Upgrading from 0.x to 1.x
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
```

Comment:
Should we basically say that you need to upgrade to 0.60 as a prerequisite? Most of the following instructions only apply to that version.

Reply:
With enabling all deprecation warnings and grepping for environment variables, upgrading to 0.60 is not a requirement. In the 1.0 release notes we make it more clear our recommendation to upgrade to 0.60. This section is more for guidance on how to migrate, whether you can upgrade to 0.60 or not.